pypeit/tests/files/spec1d_r153-J0025-0312_KASTr_20150123T025323.850.fits

```diff
@@ -1,509 +1,1051 @@
           TTYPE21 = 'BOX_COUNTS_SIG_DET'                                                  TFORM21 = '1D      '                                                            TTYPE22 = 'trace_spec'                                                          TFORM22 = '1K      '                                                            EXTNAME = 'SPAT0132-SLIT0136-DET01' / extension name                            END                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             @`&�-���@���RB�@����#�W@]'���?%h���'�@S��E��?&`���#�T@Rf���GE@R��0��?�G�K��?�'H�O�9@���@��ک k�@Q�,8*�E?0%h��;@@O��O��?0��B�T@H_Z*��@N��'�F�        @`',�Q�@���p@��+�6�@���7�O?:�m�&s�@H�w�LH�?A#�&NAT@u,�foK@B�[z`�?�vvD�E?�+��H'W@���@���O��@v�=��u?D$V��r�@D*ɂg�?I���>T@t�t��@>��g�<        @`'_Y{6@�@�z�x@��_��U@{�!f�h?:
```

Comment:
Why are these reduced data files being version controlled? Is this new or a mistake?

Reply:
Neither.  These are files used in some of the unit tests, and have been version controlled for a long time.